settings.gradle

```diff
@@ -46,6 +46,8 @@ if (workspace == null) {
   throw new GradleException("Unable to load workspace ${rootDir}/${bnd_cnf}")
 }
 
+workspace.setOffline(gradle.startParameter.offline)
```

Comment:
I don't think we need `gradle.` here. See the reference to `startParameter` below.